source/overview/features.rst

```diff
@@ -0,0 +1,282 @@
+============================
+Features Overview
+============================
+
+This page gives an overview of some of the features available in Mattermost Team and Enterprise Editions.
+
+Team Edition Features
+----------------------------------------------
+
+----------------------------------------------
+
+.. raw:: html
+
+    <table class="features__table">
+        <tbody>
+            <tr>
+                <td>
+                    <h4>1-1 and group messaging</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308655-99c4cb02-0076-11e8-90ae-bda096160995.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Chat in real time with individuals and teams. See all your conversations in one place.</p>
+                </td>
+                <td>
+                    <h4>File sharing</h4>
+
+   <img src="https://user-images.githubusercontent.com/13119842/35306887-1532c64e-006e-11e8-87fc-fb01bd5195d9.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Share documents, videos, images, and other rich media with inline previews.</p>
+                </td>
+            </tr>
+            <tr>
+                <td>
+                    <h4>Custom emojis</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308687-bc9b185c-0076-11e8-9ef6-0163486cdcc8.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Express yourself with a full set of emojis and add your own custom ones.</p>
+                </td>
+                <td>
+                    <h4>Webhooks</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35307546-26c152d8-0071-11e8-985b-f7e3e9b7ca8f.png" height="235px"  width="400px">
+
+
+.. raw:: html
+
+                    <p>Tightly integrate chat with line-of-business systems using incoming and outgoing webhooks.</p>
+                </td>
+            <tr>
+                <td>
+                    <h4>Slash commands</h4>
+
+ <img src="https://user-images.githubusercontent.com/29708087/35308696-d47206fc-0076-11e8-8df9-e333f0e24cec.png" height="235px"  width="400px">
+
+.. raw:: html
+
+                    <p>Invoke custom automation from chat - create and update tickets, kick off builds, and more.</p>
```

Comment:
"chat"

Reply:
Also these examples are very specific and some teams might not even know what they are refering to. Maybe a more generalized statement would be more effective here? 0/5

Reply:
From Chen: "Our de-facto audience is heavily DevOps, so I think technical examples are ok here - since the context is "slash commands", which is fairly technical."